Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/repository/VaccinationRepository.kt

```diff
@@ -46,33 +45,57 @@ class VaccinationRepository @Inject constructor(
     private val storage: VaccinationStorage,
     private val qrCodeExtractor: DccQrCodeExtractor,
     private val dccStateChecker: DccStateChecker,
+    private val vaccinationMigration: VaccinationMigration,
     @AppScope private val appScope: CoroutineScope,
     dscRepository: DscRepository
 ) {
 
-    private val internalData: HotDataFlow<Set<VaccinatedPerson>> = HotDataFlow(
+    private val internalData: HotDataFlow<Set<VaccinationCertificateContainer>> = HotDataFlow(
```

Comment:
This is a map in the TestCertificateRepository -> private val internalData: HotDataFlow<Map<TestCertificateContainerId, TestCertificateContainer>> = HotDataFlow(...)
which makes sure there is only one entry per container id

Reply:
So my question is, what is the reason to use a set instead?